tests/resources/expectations/models.test_llama.TestLlama2.test_model_output

```diff
@@ -1 +1 @@
-7.256865501403809e-06,0.0,0.00929248332977295,0.01114596426486969,0.0037247538566589355,0.006703168153762817,0.02313566952943802,0.03150583803653717,0.01455271989107132,0.0002584382891654968,0.03448055684566498,0.008191876113414764,0.020602405071258545,0.02093110978603363,0.020266816020011902,0.010351307690143585
\ No newline at end of file
+0.0,3.154575824737549e-05,0.009287461638450623,0.011117540299892426,0.0037216022610664368,0.006678417325019836,0.023142069578170776,0.0314808189868927,0.014560192823410034,0.0002407953143119812,0.03446689248085022,0.008167281746864319,0.02057461440563202,0.020925693213939667,0.020260170102119446,0.010330617427825928
```

Comment:
I think these results prove my hypothesis from #243: here we are NOT preserving the order anymore for the random numbers, so the results have much bigger differences (2-3 orders of magnitude more than the previous PR)